pom.xml

```diff
@@ -26,14 +26,14 @@
 
 
   <properties>
-    <revision>0.0.1-SNAPSHOT</revision>
+    <revision>0.0.0-SNAPSHOT</revision>
     <java.version>1.8</java.version>
     <global.name>JKube :: Integration Tests</global.name>
     <apache.maven.maven-compiler-plugin.version>3.8.1</apache.maven.maven-compiler-plugin.version>
     <apache.maven-failsafe-plugin.version>3.0.0-M3</apache.maven-failsafe-plugin.version>
     <apache.maven.maven-invoker.version>3.0.1</apache.maven.maven-invoker.version>
     <fabric8.kubernetes-client.version>4.6.2</fabric8.kubernetes-client.version>
-    <jkube.version>0.1-SNAPSHOT</jkube.version>
+    <jkube.version>0.1.0</jkube.version>
```

Comment:
Shouldn't it be `0.1.1-SNAPSHOT`?

I was thinking if we could get project version from build itself.
```
PROJECT_VERSION=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec)
cd it/
find . -type f -exec sed -i 's/0.1-SNAPSHOT/'"$PROJECT_VERSION"'/g' {} +
```

Reply:
Currently working on this:
https://github.com/manusa/jkube/commit/3f2791e67d04d75dd8754bcf622682ab8da00d49

Reply:
Jkube version in POM should be ignored when run on CI

Reply:
ah, sorry :smile:

Reply:
Yeah, right. But I thought you were plannig to run it on released version.

Reply:
Ok. You have to think that IT are hostedin a different repo. So that repo needs access to sth that's actually in Maven central (or any other repository), so in the scope of **this** project (jkube-integration-tests) we must reference at least the latest stable version (or published snapshot).
Anyway, these tests will never be run in the scope of this repo from a CI. So whenever a developer is working on the IT project, he/she should be aware of what to test either by changing pom version manually or by using -Djkube.version CLI argument